manifests/cf-manifest/scripts/rotate-vars-store-secrets.rb

```diff
@@ -75,10 +78,13 @@ def rotate(manifest, vars_store,
            passwords: false,
            rsa: false,
            ssh: false,
+           vars_to_rotate: nil,
            vars_to_preserve: [],
            delete: false)
 
-  vars = manifest.fetch("variables").reject { |v| vars_to_preserve.include?(v["name"]) }
+  vars = manifest.fetch("variables")
+  vars = vars.select { |v| vars_to_rotate.include?(v["name"].gsub(/_old$/, '')) } unless vars_to_rotate.nil?
```

Comment:
shouldn't we throw an error if a variable ends with `_old` instead of silently replacing it?

Reply:
That code picks any variable that is defined in the `variables` that is in the list of `vars_to_rotate`, and also the corresponding `_old` if exists.

For instance:

```
vars= [{'name'=>'one'},{ 'name'=>'two'}, {'name'=>'one_old'}]
vars_to_rotate=['one']

vars = vars.select { |v| vars_to_rotate.include?(v["name"].gsub(/_old$/, '')) } unless vars_to_rotate.nil?
```

That picks: `[{"name"=>"one"}, {"name"=>"one_old"}]`

Reply:
Ah ok, I understand now, thanks.